### PR TITLE
Fix typos in pkg/volume/azure_dd

### PR DESCRIPTION
**Pull request by liubin**

### CODE REVIEW

**Comment by feiskyer** [comment]
@liubin Thanks for fixing those typos

File: pkg/volume/azure_dd/attacher.go [hunk 1]
```diff
@@ -181,7 +181,7 @@ func (a *azureDiskAttacher) WaitForAttach(spec *volume.Spec, devicePath string,
 
 		// did we find it?
 		if newDevicePath != "" {
-			// the curent sequence k8s uses for unformated disk (check-disk, mount, fail, mkfs.extX) hangs on
+			// the current sequence k8s uses for unformated disk (check-disk, mount, fail, mkfs.extX) hangs on
```
Comment (nikhita) [inline pkg/volume/azure_dd/attacher.go]: This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-86f2cf81f39b855ad72e3fac35b08249. Closing.

/close

**Pull request closed without merging**